virt/virt-4-10-release-notes.adoc

```diff
@@ -284,3 +284,9 @@ $ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged 'networka
 
 * The web console does not display virtual machine templates that are deployed to a custom namespace. Only templates deployed to the default namespace display in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2054650*])
 ** As a workaround, avoid deploying templates to a custom namespace.
+
+* If a VMI has the `spec.evictionStrategy` field set to `LiveMigrate`, then you cannot migrate or evict the VMI or upgrade a cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2073880[*BZ#2073880*])
+** There are two workaround options:
+*** Remove the `spec.evictionStrategy` field from the VM declaration.
+
+*** Manually stop the VM before starting an {VirtProductName} upgrade.
```

Comment:
This is technically not accurate. The running VM will block eviction of the node, thus is will block an OpenShift upgrade. Upgrading OpenShift Virtualization by itself does not evict the node.

Does the distinction make sense?

------

*** Manually stop the VM before starting an OpenShift upgrade.